package.json

```diff
@@ -13,5 +13,9 @@
   },
   "keywords": [],
   "author": "",
-  "license": "AGPL-3.0-only"
-}
\ No newline at end of file
+  "license": "AGPL-3.0-only",
+  "dependencies": {
```

Comment:
move the dependencies to the package.json file inside the "frontend" folder

Reply:
ok